moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -404,6 +399,10 @@ void RobotStateDisplay::onDisable()
 void RobotStateDisplay::update(float wall_dt, float ros_dt)
 {
   Display::update(wall_dt, ros_dt);
+
+  if (load_robot_model_)
+    loadRobotModel();
+
```

Comment:
+1, I have an Rviz config file that loads **11** URDFs for different displays, and its quite slow showing the GUI